content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -92,6 +92,18 @@ Common forms of values include:
 
 One of the [recommended labels](/docs/concepts/overview/working-with-objects/common-labels/#labels).
 
+### kubectl.kubernetes.io/restartedAt
+
+Example: `kubectl.kubernetes.io/restartedAt: "2023-03-31T15:30:00Z"`
+
+Used on: Pod
+
+In this example, the annotation indicates that the Pod was most recently restarted at 3:30 PM UTC on March 31st, 2023.
```

Comment:
I don't think this is required. However, I'm not against its addition.

Reply:
so what's your thoughts on this, i will remove these lines or leave as it is!

Reply:
```suggestion
In this example, the annotation indicates that the Pod is part of a workload where the last triggered restart
occurred at 15:30 (UTC) on March 31st, 2023.
```